Python/Cryptography/Strong Password.py

```diff
@@ -0,0 +1,38 @@
+import math
+import os
+import random
+import re
+import sys
+
+
+def minimumNumber(n, password):
+    nums = "0123456789"
+    lower_case = "abcdefghijklmnopqrstuvwxyz"
+    upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    special = "!@#$%^&*()-+"
+    counts = 4
+    if any(list(map(lambda x: x in nums,password))):

```

Comment:
Please add the purpose of each if-statement

Reply:
Explain each and every case OR edge condition that the if-statements are dealing